src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -1597,9 +1628,7 @@ bool OIIO_API convolve (ImageBuf &dst, const ImageBuf &src, const ImageBuf &kern
 /// Laplacian kernel,
 ///
 ///                     [ 0  1  0 ]
-///                     [ 1 -4  1 ]
 ///                     [ 0  1  0 ]
-///
 ImageBuf OIIO_API laplacian (const ImageBuf &src, ROI roi={}, int nthreads=0);
```

Comment:
Whoa, what's up with this?

Reply:
Oops, my bad, I don't know what happened there. Fixed now.